chains/ethereum/src/utils/wagmiHelpers.ts

```diff
@@ -18,8 +18,49 @@ export function getClient() {
   return client
 }
 
-function onConnectorChange(event: ConnectorData) {
-  if (event.account) AccountCtrl.setAddress(event.account)
+async function getBalance(account: string) {
+  const opts: GetBalanceOpts = {
```

Comment:
```suggestion
  const opts = {
```

Let typescript do the work